File: Lab-1/1_1/src/MySquare.java
// Java Program by Alex Bledsoe, 04/06/2017
class MySquare {
    private double sideLength;

    double getArea() {
        return sideLength * sideLength;
    }

    double getSideLength() {
        return sideLength;
    }

    void setSideLength(double l) {
        sideLength = l;
    }
}

File: KotlinConversion/src/Lab_1/1_1/MySquare.kt
package Lab_1

/**
 * Created by Alex Bledsoe, 6/2/2017.
 */
class MySquare(input: Double) {
    var length: Double = input
    fun getArea(): Double {
        return length * length
    }
    fun getSideLength(): Double {
        return length
    }
}